dev-docs/external_dependency.md

```diff
@@ -0,0 +1,93 @@
+#External Dependecies
+
+1. Elastic search
+   * automate-elasticsearch
+     * Version: 6.8.14
+
+2. ngnix
+   * automate-es-gateway 
+   * automate-load-balancer
+     * Version: 1.19.2 
+  
+3. Open JDK(Java) binaries
+   * automate-openjdk
+     * Version: 11.0.10+9
+
+4. Memecached
+   * automate-builder-memcached
+     * Version: 1.5.19
+
+5. bookshelf
+   * automate-cs-bookshelf
+     * Version: 14.4.4/20210520120637
+
+6. builder-api
+   * automate-builder-api
+     * Version:9034/20200827185135 
+
+7. builder-api-proxy
+   * automate-builder-api-proxy
+     * Version: 8997/20200812161534
+
+8. curl
+   * automate-cs-nginx
+     * Version: 7.68.0/20200601114640
+
+9. ruby
+   * automate-cs-nginx
+     * Version: 2.6.5/20200404043345
+
+10. oc_bifrost
+   * automate-cs-oc-bifrost
+     * Version: 14.4.4/20210520120641
+
+11. oc_erchef
+   * automate-cs-oc-erchef
+     * Version: 14.4.4/20210520120641
+
+12. Dex
+   * automate-dex
+     * Version: 2.27.0
+
+13. postgresql
+   * automate-postgresql
+     * Version: 9.6.11
+
+   * pg-sidecar-service
+     * Version: 9.6.11/20190305213451
+
+14. go
+   * automate-scaffolding-go
+     * Version: 1.14
+
+15. UI dependencies
+   * components/automate-ui/package.json
+    
+
+16. openresty-noroot
+   * automate-workflow-nginx
+     * Version: 1.13.6.2/20210520120637
+
+17. inspec
+   * compliance-service
+     * Version: 4.24.32/20210112215549
+
+18. elixir
+   * notifications-service
+     * Version: 1.8.0
+
+19. postgresql-client
+   * pg-sidecar-service
+     * Version: 9.6.11/20190305213520 
+
+20. sqitch_pg
+   * pg-sidecar-service
+     * Version: 3.7.4/20190305225225
+
+21.  https://github.com/chef/automate/blob/master/go.mod
```

Comment:
Please add links with some context and in proper MD link form.

Reply:
done